eda-modbus-bridge/CHANGELOG.md

```diff
@@ -1,11 +1,17 @@
 # Change log
 
-This change log is only for the Home Assistant add-on. See the 
-[eda-modbus-bridge CHANGELOG](https://github.com/Jalle19/eda-modbus-bridge/blob/master/CHANGELOG.md) for changes to the 
+This change log is only for the Home Assistant add-on. See the
+[eda-modbus-bridge CHANGELOG](https://github.com/Jalle19/eda-modbus-bridge/blob/master/CHANGELOG.md) for changes to the
 application itself.
 
 The version number here doesn't necessarily reflect `eda-modbus-bridge`'s version numbering.
 
 ## 1.0.0
 
 Initial release as a proper Home Assistant add-on!
+
+## 1.0.1
```

Comment:
Change logs are normally "newest first", we can probably change the order the next time we make a change?